src/App.js

```diff
@@ -1,85 +1,107 @@
 import React from 'react';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
+import { NewTodo } from './components/NewTodo/NewTodo';
+import { todosFromServer } from './components/api/todosFromServer';
 
-function App() {
-  return (
-    <section className="todoapp">
-      <header className="header">
-        <h1>todos</h1>
+class App extends React.Component {
+  state = {
+    todos: [...todosFromServer],
+    filterProp: 'all',
```

Comment:
зачем приставка `prop`?
```suggestion
    filterField: 'all',
```